iocore/net/P_SSLNetVConnection.h

```diff
@@ -260,6 +260,17 @@ class SSLNetVConnection : public UnixNetVConnection
   /// Set by asynchronous hooks to request a specific operation.
   SslVConnOp hookOpRequested;
 
+  bool
+  getClientVerifyEnable()
+  {
+    return clientVerifyEnable;
+  }
+  void
+  setClientVerifyEnable(int enable)
```

Comment:
Well, if these APIs are changing in a subsequent PR, I strongly suggest you combine these into one. I still don't see why it needs to be a MgmtInt, and definitely getClientVerifyEnable() is wrong if you expect it to contain something other than 0|1.

I think you have to do one of two things:

1) Make this PR correct (correct types etc.), and then deal with future changes later. I.e. don't mix behavior in this PR with future expected behavior.

2) Combine this with your next PR, but I'm slightly concerned about changing the behavior of this existing commit in a minor release.